db/fixtures/classifications.yml

```diff
@@ -943,3 +943,41 @@
   :parent_id: 0
   :default: true
   :single_value: "1"
+- :description: Transform VM
```

Comment:
This tag category will be than used also for network and storage mapping, we should use a bit more generic term like:
"Transformation".